nbs/src/core/models.ipynb

```diff
@@ -42,7 +42,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    class AutoTBATS(_TS):
</pre><p>The methods here are pretty much the same than for TBATS, right? I believe the only differences are in the default parameters. We can probably define TBATS first, inherit from it (<code>class AutoTBATS(TBATS)</code>) and just override the <code>__init__</code>  method.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/759/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>

Reply:
Agreed, AutoTBATS should be a subclass of TBATS.